podpac/core/data/test/test_wcs.py

```diff
@@ -1,196 +1,145 @@
-        podpac.core.data.ogc.requests = None
 
-        # no requests, urllib3
-        podpac.core.data.ogc.urllib3 = None
+@pytest.mark.integration
+class TestWCSIntegration(object):
+    source = "https://maps.isric.org/mapserv?map=/map/sand.map"
 
-        node = WCS(source=self.source)
-        with pytest.raises(Exception):
-            node.wcs_coordinates
+    def setup_class(cls):
+        cls.node1 = WCSBase(source=cls.source, layer="sand_0-5cm_mean", format="geotiff_byte")
 
-        # put all dependencies back
-        podpac.core.data.ogc.requests = requests
-        podpac.core.data.ogc.urllib3 = urllib3
-        podpac.core.data.ogc.lxml = lxml
+        cls.node2 = WCS(source=cls.source, layer="sand_0-5cm_mean", format="geotiff_byte")
 
     def test_coordinates(self):
-        """get coordinates"""
-
-        self.mock_requests()
-        node = WCS(source=self.source)
-
-        # equal to wcs coordinates when no eval coordinates
-        coordinates = node.coordinates
-        wcs_coordinates = node.wcs_coordinates
-        assert coordinates == wcs_coordinates
-
-        # with eval coordinates
-        # TODO: use real eval coordinates
-        node._output_coordinates = coordinates
-        coordinates = node.coordinates
-
-        assert isinstance(coordinates, Coordinates)
-        # TODO: one returns monotonic, the other returns uniform
-        assert coordinates == node._output_coordinates
-        assert coordinates["lat"]
-        assert coordinates["lon"]
-        assert coordinates["time"]
-
-    def test_get_data(self):
-        """get data from wcs server"""
-
-        self.mock_requests()
-        node = WCS(source=self.source)
-        lat = node.coordinates["lat"].coordinates
-        lon = node.coordinates["lon"].coordinates
-        time = node.coordinates["time"].coordinates
-
-        # no time
-        notime_coordinates = Coordinates(
-            [clinspace(lat[0], lat[-2], 10), clinspace(lon[0], lon[-2], 10), "2006-06-14T17:00:00"],
-            dims=["lat", "lon", "time"],
-        )
-
-        with pytest.raises(ValueError):
-            output = node.eval(notime_coordinates)
-            assert isinstance(output, UnitsDataArray)
-            assert output.coordinates["lat"][0] == node.coordinates["lat"][0]
-
-        # time
-        time_coordinates = Coordinates(
-            [clinspace(lat[0], lat[-2], 10), clinspace(lon[0], lon[-2], 10), clinspace(time[0], time[-1], len(time))],
-            dims=["lat", "lon", "time"],
-        )
-
-        with pytest.raises(ValueError):
-            output = node.eval(time_coordinates)
-            assert isinstance(output, UnitsDataArray)
-
-        # requests exceptions
-        self.mock_requests(data_status_code=400)
-        with pytest.raises(Exception):
-            output = node.eval(time_coordinates)
-        with pytest.raises(Exception):
-            output = node.eval(time_coordinates)
+        self.node1.coordinates
+
+    def test_eval_grid(self):
```

Comment:
Could you add a test with a different CRS (one not supported by the remote server).